docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -424,10 +472,58 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+Here is a example of the command to run :
+
+```bash
+sudo iptables -A INPUT -s 123.123.123.123 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
+You can verify that your signer node is correctly configured and registered by checking your signer node logs or by running this command :
+
+```bash
+if grep -q "STATE MACHINE: new cycle: Registered" **YOUR_SIGNER_NODE_LOGS_PATH**; then
+    echo ">> Your signer node is registered"
+else
+    echo ">> Your signer node is not registered"
+fi
+```
+
```

Comment:
This should be a separate section below the existing tip box:
### Verify that your signer is registered

Also:
- The command should probably first write to a file, chmod the file and the use it for clarity
- Add a tip box for the associated example command